src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -79,16 +79,16 @@ public NewAddressModel GetNewAddress()
         /// RPC method that returns the spendable balance of all accounts.
         /// Uses the first wallet and account.
         /// </summary>
+        /// <param name="minConfirmations">Only include transactions confirmed at least this many times. (default=1)</param>
+        /// <param name="isWatchOnly">Also include balance in watch-only addresses. (not used yet)</param>
         /// <returns>Total spendable balance of the wallet.</returns>
         [ActionName("getbalance")]
         [ActionDescription("Gets wallets spendable balance.")]
-        public decimal GetBalance()
+        public decimal GetBalance(int? minConfirmations, bool? isWatchOnly)
```

Comment:
isWatchOnly isn't used anywhere

Reply:
maybe just default minConfirmations to 1?

Reply:
Is watchOnly needed for compatibility?
I prefer the signature `public decimal GetBalance(int minConfirmations = 1, bool isWatchOnly = false)`